app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/BuyerOrdersActivity.java

```diff
@@ -0,0 +1,62 @@
+package com.epfl.neighborfood.neighborfoodandroid.ui.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.epfl.neighborfood.neighborfoodandroid.NeighborFoodApplication;
+import com.epfl.neighborfood.neighborfoodandroid.R;
+import com.epfl.neighborfood.neighborfoodandroid.adapters.BuyerOrderListAdapter;
+import com.epfl.neighborfood.neighborfoodandroid.adapters.VendorOrderListAdapter;
+import com.epfl.neighborfood.neighborfoodandroid.models.Meal;
+import com.epfl.neighborfood.neighborfoodandroid.models.Order;
+import com.epfl.neighborfood.neighborfoodandroid.models.User;
+import com.epfl.neighborfood.neighborfoodandroid.ui.viewmodels.BuyerOrdersActivityViewModel;
+import com.epfl.neighborfood.neighborfoodandroid.ui.viewmodels.factories.BuyerOrdersViewModelFactory;
+import com.epfl.neighborfood.neighborfoodandroid.util.Triplet;
+
+import java.util.ArrayList;
+
+public class BuyerOrdersActivity extends AppCompatActivity {

```

Comment:
I think this activity should better be named BuyerOrdersList, or something of that sort, to have it more differetiated between the the BuyerOrderDetails